docs/user/README.md

```diff
@@ -0,0 +1,40 @@
+---
+title: BTP Operator Module
+---
+
+
+## Overview
+
+Within the BTP Operator module, [BTP Manager](../../README.md) installs [SAP BTP Service Operator](https://github.com/SAP/sap-btp-service-operator/blob/main/README.md).
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- Kubernetes cluster (you can use [k3d](https://k3d.io)) 
+
+## Enable BTP Operator module
+
+To enable the BTP Operator module, use the following command:
+
+```
+kubectl apply -f https://github.com/kyma-project/btp-manager/releases/latest/download/btp-manager.yaml
+```
+Use the same command to upgrade the module to the latest version.
+
+For more details on other installation options, read the [Install and uninstall BTP Manager](../contributor/01-01-installation.md) document.
+
+## How BTP Operator module works
+
+BTP Manager provisions, updates, and deprovisions SAP BTP Service Operator along with its resources, Service Instances, and Service Bindings. SAP BTP Service Operator manages SAP BTP services in your cluster.
+
+Read [BTP Manager operations](../contributor/02-01-operations.md) and the [SAP BTP Service Operator](https://github.com/SAP/sap-btp-service-operator) documentation to learn more.
```

Comment:
I'm wondering if we should link from the `user` folder to the `contributor` folder :thinking:. Theoretically yes, but in practice, wouldn't that confuse the end-user?

Reply:
I think we can do it as users might want to know more. The link takes them to extra info on the subject. I'll rephrase the sentence to make it clearer.